share/spice/mars_photos/mars_photos.js

```diff
@@ -0,0 +1,33 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_mars_photos = function(api_result){
+        if (!api_result || api_result.error) {
```

Comment:
Probably good to check if `photos` exists and if it has a length > 0:

![screen shot 2016-01-17 at 2 11 02 pm](https://cloud.githubusercontent.com/assets/81969/12379246/362826ee-bd24-11e5-8c18-9e45633b447a.png)